inc/integrations/class-form-email.php

```diff
@@ -162,6 +162,24 @@ public function build_error_email( $error, $form_data ) {
         return ob_get_clean();
     }
 
+	/**
+	 * @param string $error The error message.
+	 * @param Form_Data_Request $form_data The form request data.
+	 * @return false|string
+	 */
+	public function build_error_body( $error, $form_data ) {
+		ob_start();  ?>
+		<h3><?php esc_html__( 'An error has occurred when a user submitted the form.', 'otter-blocks' ) ?></h3>
+		<div style="padding: 10px;">
+			<span style="color: red;"> <?php esc_html__( 'Error: ', 'otter-blocks' ) ?> </span>
+			<?php esc_html__( $error, 'otter-blocks' ) ?>
+			<br/>
+			<p> <?php esc_html__( 'Please check your Form credential from the email provider.', 'otter-blocks' ) ?></p>
+		</div>
+		<?php
+		return ob_get_clean();
+	}
+
```

Comment:
@HardeepAsrani, any idea why this is giving no string? Using this method to render the HTML is unreliable; sometimes it works, sometimes it is not.